spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -41,7 +41,7 @@
   let(:trace_id) { SecureRandom.uuid }
   let(:user) { build(:user, :signed_up) }
   let(:threatmetrix_session_id) { SecureRandom.uuid }
-  let(:threatmetrix_request_id) { '1234' }
+  let(:threatmetrix_request_id) { 'ddp-mock-transaction-id-123' }
```

Comment:
let's use the mocked constant here:
```suggestion
  let(:threatmetrix_request_id) { Proofing::Mock::DdpMockClient::TRANSACTION_ID }
```